index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio</title>
+    <title>Miliyon-Ayalew | Portfolio</title>
```

Comment:
Added descriptive title to our project in order to implement the best practice of first review web accessibility.